app/views/grandmothers/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Grandmothers new</h1>
+
+<%= simple_form_for @grandmother do |f| %>
+  <%= f.input :first_name %>
+  <%= f.input :last_name %>
+  <%= f.input :age %>
+  <%= f.input :nationality %>
+  <%= f.input :description %>
```

Comment:
We need to talk with the guys but maybe the description part, could be like this: 
<%= f.input :description, as: :text %>